docker/docker-entrypoint.sh

```diff
@@ -152,11 +157,6 @@ if [ "$(id -u)" == "$(id -u paperless)" ]; then
 	gosu_cmd=()
 fi
 
-# Install additional languages if specified
-if [[ -n "$PAPERLESS_OCR_LANGUAGES" ]]; then
-	install_languages "$PAPERLESS_OCR_LANGUAGES"
-fi
-
 initialize
```

Comment:
Why not move this up a few lines instead?

Reply:
I put it there to avoid changing the chronology, the package are still installed prior `initialize` but after sourcing ``env-from-file.sh`

Since I don't know the codebase well enough I though that the order matter

Reply:
There is this comment 

```
	# Leave this last thing
	custom_container_init
```
https://github.com/paperless-ngx/paperless-ngx/blob/2006a5e9b52e23dd3ff2950376fa33b805ffb053/docker/docker-entrypoint.sh#L116C25-L117C2